src/labels/CustomLabels.labels

```diff
@@ -1617,6 +1617,14 @@
         <shortDescription>Error when the functionality is not supported for Elevate Recurring Donation.</shortDescription>
         <value>This functionality is currently not supported for Recurring Donations generated from Elevate Payment Services.</value>
     </labels>
+    <labels>
+        <fullName>RD2_ElevatePendingStatus</fullName>
```

Comment:
@sam-knox please review the 2 new custom labels for cancel commitment. `RD2_ElevatePendingStatus` is set on the Status Reason field when the callout is happening to cancel commitment. Once the callout is completed, The field will swap back to the original close reason.

Reply:
@andrewyu-salesforce - How long does the callout process take? I'm assuming that a user would only see `RD2_ElevatePendingStatus` very briefly. Or can it take some time to complete?

Reply:
It depends on when the queue run. It will be maximum 10 seconds per record after the async job kick off. I will say it will be very briefly.